custom/enikshay/reports/worker_activity.py

```diff
@@ -0,0 +1,87 @@
+from corehq.apps.es import UserES
+from corehq.apps.es.users import mobile_users
+from corehq.apps.reports.analytics.esaccessors import get_submission_counts_by_user
+from corehq.apps.reports.datatables import DataTablesHeader, DataTablesColumn
+from corehq.apps.reports.filters.dates import DatespanFilter
+from corehq.apps.reports.standard import CustomProjectReport
+from corehq.apps.reports.standard.monitoring import WorkerActivityReport
+from corehq.apps.reports.util import get_simplified_users
+from custom.enikshay.reports.filters import EnikshayLocationFilter
+
+from django.utils.translation import ugettext as _
+
+from dimagi.utils.decorators.memoized import memoized
+
+
+class EnikshayWorkerActivityReport(WorkerActivityReport, CustomProjectReport):
+    slug = 'enikshay_worker_activity_report'
+    num_avg_intervals = 4
+    ajax_pagination = True
+    is_cacheable = False
+
+    @property
+    def fields(self):
+        return DatespanFilter, EnikshayLocationFilter
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn(_("User")),
+            DataTablesColumn(_("# Forms Submitted"), sortable=False),
+            DataTablesColumn(_("Avg # Forms Submitted"), sortable=False),
+            DataTablesColumn(_("Last Form Submission"), sortable=False)
+        )
+
+    @property
+    def locations_id(self):
+        return EnikshayLocationFilter.get_value(self.request, self.domain)
+
+    @property
+    def user_query(self):
+        user_query = UserES().domain(self.domain).filter(mobile_users())
+        location_ids = self.locations_id
```

Comment:
nit: weird that this is called `location_id` and then you set it to var called `location_ids`